docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/cthp/paragraph--cgov-cthp-guide-card.html.twig

```diff
@@ -6,7 +6,7 @@
       {{ content.field_cthp_guide_card_desc }}
     {% endif %}
 
-    {% if content.field_cthp_pdq_link_heading|field_value or content.field_cthp_pdq_links["#items"] %}
+    {% if content.field_cthp_pdq_link_heading|field_value and content.field_cthp_pdq_links["#items"] %}
       <h4 class="cthp-pdq-label">
```

Comment:
If both fields need the content.field_cthp_pdq_links["#items"] to be true, why not just wrap both in the one conditional and only the heading field in the other?

Reply:
good point.  fixing now.